gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/ImportCRDUseCase.java

```diff
@@ -48,6 +49,7 @@
  * @author Antoine CORDIER (antoine.cordier at graviteesource.com)
  * @author GraviteeSource Team
  */
+@TransactionalUseCase
```

Comment:
What about a `@UseCase(transactional = true)` ?

Reply:
I think I will remove the `@TransactionalUseCase` and only keep `@UseCase`. The former annotation were introduce to "enable" Spring transaction. I suppose we can apply `@Transactional` annotation on all UseCase